src/deluge/model/settings/runtime_feature_settings.cpp

```diff
@@ -59,6 +59,28 @@ static void SetupOnOffSetting(RuntimeFeatureSetting& setting, const deluge::stri
 	};
 }
 
+static void SetupSyncScalingActionSetting(RuntimeFeatureSetting& setting, char const* const displayName,
+                                          char const* const xmlName, RuntimeFeatureStateSyncScalingAction def) {
+	setting.displayName = displayName;
+	setting.xmlName = xmlName;
+	setting.value = static_cast<uint32_t>(def);
+
+	setting.options[0] = {
+	    .displayName = "SCAL",
+	    .value = RuntimeFeatureStateSyncScalingAction::SyncScaling,
+	};
+
+	setting.options[1] = {
+	    .displayName = "FILL",
+	    .value = RuntimeFeatureStateSyncScalingAction::Fill,
+	};
+
+	setting.options[2] = {
+	    .displayName = NULL,
+	    .value = 0,
+	};
+}
```

Comment:
How does this look as a way to allow users to assign a specific front panel  button as FILL?

Reply:
https://discord.com/channels/608916579421257728/1118654563596128336/1139997858028867667